src/lib/utils/conversion.ts

```diff
@@ -0,0 +1,5 @@
+import { LAMPORTS_PER_SOL } from '@solana/web3.js';
```

Comment:
Can we move this to `src/modules/sol` can use that module to house all sol related utility functions. I want to stay way from a general lib utils blackhole.

Reply:
Sure!